CMEPS-interface/CMakeLists.txt

```diff
@@ -62,8 +69,103 @@ list(APPEND _mediator_files
   CMEPS/mediator/med_phases_post_rof_mod.F90
   CMEPS/mediator/med_phases_post_wav_mod.F90)
 
-### Create target library and set PUBLIC interfaces on the library
-add_library(cmeps STATIC ${_nems_util_files} ${_mediator_files})
+###############################################################################
+### Host model for CCPP
+###############################################################################
+
+### Query FV3 path
+get_filename_component(FV3_PATH "../FV3" REALPATH)
+
+### If FV3 is available build CCPP host model under CMEPS too
+if(EXISTS ${FV3_PATH})
+  ### Print out information
+  message("FV3_PATH = ${FV3_PATH}")
+  message("Compiling CMEPS with CCPP support.")
+
+  ### Set environment variable for FV3 directory, used by config file
+  set(ENV{FV3_PATH} ${FV3_PATH})
+
+  ### Look for CMEPS CCPP suites and get list of them to provide CCPP preprocessor
+  set(_ccpp_suites_arg "--suites=")
+  FILE(GLOB FILES "CMEPS/ufs/ccpp/suites/*.xml")
+  foreach(MYFILE ${FILES})
```

Comment:
Does this mean the CMEPS will compile with all the suite files under CMEPS/ufs/ccpp/suites?

Reply:
There are good reasons for not using `glob` in `cmake` (including security reasons), therefore I would recommend hardcoding the names of the suites if you don't want to add the complexity of additional compile time options.

Reply:
@climbfuji This is fixed. I remove the glob and make suite name static.